package.json

```diff
@@ -137,6 +137,7 @@
     "analytics-node/axios": "^0.21.2",
     "bn.js": "^5.2.1",
     "ganache-core/lodash": "^4.17.21",
+    "ganache/abstract-level": "1.0.4",
```

Comment:
added this entry in the resolutions as per this suggestion from @legobeat 
https://github.com/MetaMask/metamask-extension/pull/26612#discussion_r1728234068